README.md

```diff
@@ -1,197 +1,33 @@
-# Improve School Workload and Wellbeing
+# Improve school workload and wellbeing
```

Comment:
It should now be 'Improve workload and wellbeing for school staff'

we had to change names just before you joined Abi.